content/applications/finance/accounting/bank/feeds/bank_statements.rst

```diff
@@ -2,11 +2,11 @@
 Bank statements
 ===============
 
-Importing your bank statements allows you to keep track of your bank account transactions and
-reconcile them with the ones recorded in Odoo.
+Importing bank statements allows keeping track of bank account transactions and reconcile them with
+the ones recorded in Odoo.
 
-We recommend using :doc:`the bank synchronization feature <bank_synchronization>` for more
-efficiency. However, if you do not want to use it or if your bank is not yet supported, other
+With :doc:`the bank synchronization feature <bank_synchronization>` the process is going to be
+automated. However, if you do not want to use it or if your bank is not yet supported, other
 options exist:
 
 - Import the bank statement files delivered by your bank;
```

Comment:
- Import bank statement files;
- Register bank transactions manually.
- --> I would take the title of the sections below and add custom anchors.